client/app/views/contact.coffee

```diff
@@ -167,11 +168,17 @@ module.exports = class ContactView extends ViewCollection
 
             @model.setN @contactName.getStructuredName()
             @model.set note: @notesfield.val()
+            @model.set 'bday', @$('.bday-input').val()
+            @model.set 'org', @$('.org-input').val()
+            @model.set 'title', @$('.title-input').val()
+            @model.set 'department', @$('.department-input').val()
+            @model.set 'url', @$('.url-input').val()
```

Comment:
I think url can be used as a simple field. I think it's Google Contacts or Android 4.4 who made me change that.